src/SelectLocation.js

```diff
@@ -87,7 +77,8 @@ export default class SelectLocation extends Component {
       initialLoad: false,
       zoom: "",
       viewLocation: [],
-      workOrderDetailsViewer: false
+      workOrderDetailsViewer: false,
```

Comment:
Not a big deal because its not related to this PR that much but the naming convention in JS for boolean typed variables is to start with `is` or `has`.  Here seems like `isWorkOrderDetailsViewer` would make the most sense.

https://www.samanthaming.com/tidbits/34-better-boolean-variable-names

Reply:
Good to know - I like learning about self-documenting code and this isn't a convention that I was familiar with. Thanks!